environment_validation.py

```diff
@@ -26,18 +27,17 @@ def validate_environment_settings(
         )
 
     # Check the server private key
-    if not SERVER_PRIVATE_KEY.exists() or SERVER_PRIVATE_KEY.is_dir():
-        raise ValueError(
-            f"The specified server private key file {SERVER_PRIVATE_KEY} does not "
-            "exist or is a directory. Please provide a valid path."
-        )
+    if not SERVER_PRIVATE_KEY.exists() :
+        raise ValueError(f"The specified server private key file {SERVER_PRIVATE_KEY} does not exist.")
+    elif SERVER_PRIVATE_KEY.is_dir():
+        raise ValueError(f"The specified server private key file {SERVER_PRIVATE_KEY} is a directory.")
 
     # Check the server public key
-    if not SERVER_PUBLIC_KEY.exists() or SERVER_PUBLIC_KEY.is_dir():
-        raise ValueError(
-            f"The specified server public key file {SERVER_PUBLIC_KEY} does not exist "
-            "or is a directory. Please provide a valid path."
-        )
+
+    if not SERVER_PUBLIC_KEY.exists() :
+        raise ValueError(f"The specified server public key file {SERVER_PUBLIC_KEY} does not exist.")
+    elif SERVER_PUBLIC_KEY.is_dir():
```

Comment:
the elif is wrong. a path OR file will return True for .exists() so this line will never be executed. you need to make it a standalone if

https://docs.python.org/3/library/os.path.html

Reply:
done